### ISSUE TITLE
[JavaScript] possible issue with constructor syntax

### ISSUE THREAD

**Issue by jcberquist** [sublimehq/Packages#198]
With the current `literal-constructor` context any trailing spaces after a constructor or any characters found in the look-ahead will get the scope `meta.instance.constructor`. For instance in `new Date().getTime()` the whole statement gets the `meta.instance.constructor` scope. I would have thought that `.getTime()` should not be part of that scope? If I am right a possible fix would be to change the line
```
push: function-call-params
```
to 
```
set:
  - meta_scope: meta.instance.constructor
  - include: function-call-params
```
so that the constructor scope pops off after the parenthesis. Also, is there a reason it is not `meta.instance.constructor.js`?

**Comment by wbond** [comment]
These should be fixed via ed89b19a05a2d9ad8a1584b30f75e833aa930728

**Comment by jcberquist** [comment]
Thanks again! Looking forward to the next dev build :smile: The parsing speed of your syntax is amazing and I love all of the meta scopes you have added!
